ONSdigital/dp-design-system
# Change query selector for search

### What

This fixes an issue not working when viewing search and aggregation pages with Welsh language

### How to review

- Pull this branch
- Run dp-frontend-search-controller using make debug ENABLE_REWORKED_DATA_AGGREGATION_PAGES=true
- Port forward the api router to sandbox dp ssh sandbox web 1 -p 23200:localhost:10800
- Head over to:
  - http://cy.localhost:25000/search?q=climate
  - http://cy.localhost:25000/alladhocs
  - http://cy.localhost:25000/datalist
  - http://cy.localhost:25000/publishedrequests
  - http://cy.localhost:25000/staticlist
  - http://cy.localhost:25000/topicspecificmethodology
  - http://cy.localhost:25000/timeseriestool
  - http://cy.localhost:25000/publications
  - http://cy.localhost:25000/allmethodologies

and play around with pagination, it should now scroll to the top when clicking on each one.
You can also try it in English

### Who can review

Frontend
